docs/api/ReactWrapper/slice.md

```diff
@@ -0,0 +1,44 @@
+# `.slice([begin[, end]]) => ReactWrapper`
+
+Returns a new wrapper with a subset of the nodes of the original wrapper, according to the rules of `Array#slice`.
+
+
+#### Arguments
+
+1. `begin` (`Number` [optional]): Zero-based index from which to slice (defaults to `0`).
+2. `end` (`Number` [optional]): Zero-based index at which to end slicing (defaults to `length`).
```

Comment:
(also, please use `1.` for every item in any numbered list, so that reordering doesn't require renumbering)